understanding/22/target-size-minimum.html

```diff
@@ -50,74 +50,104 @@ <h2>Intent of Target Size (Minimum)</h2>
+          </ol>
+        </section>
 
-        <p>The requirement is independent of the zoom factor of the page; when users zoom in the CSS pixel size of elements does not change. This means that authors cannot meet it by claiming that the target will have enough spacing or sufficient size if the user zooms into the page.</p>
-      
-        <p>The requirement does not apply to targets while they are obscured by content displayed as a result of a user interaction or scripted behavior of content, for example:</p>
+        <section id="size-requirement">
+          <h3>Size requirement</h3>
 
-        <ul>
-          <li>interacting with a combobox shows a dropdown list of suggestions</li>
-          <li>activating a button displays a modal dialog</li>
-          <li>content displays a cookie banner after page load</li>
-          <li>content displays a "Take a survey" dialog after some period of user inactivity</li>
-        </ul>    
-        <p>The requirement does however apply to targets in any new content that appears on top of other content.</p>
-        
-        <p>While the Success Criterion primarily helps touch users by providing target sizing to prevent accidental triggering of adjacent targets, it is also useful for mouse or pen users. It reduces the chances of erroneous activation due to either a tremor or reduced precision, whether because of reduced fine motor control or input imprecision.</p>
+          <p>For a target to be "24 by 24 CSS pixels", it must be possible to draw a 24 by 24 CSS pixel (px) square such that the square is completely within the target.</p>
 
-        <h3>Spacing versus size</h3>
-        
-        <p>Spacing, alone and in conjunction with size, can improve user experience. There is less chance of accidentally activating a neighboring target if a target is not immediately adjacent to another. Touchscreen devices and user agents generally have internal heuristics to identify which link or control is closest to a user's touch interaction - this means that sufficient spacing around a target can work as effectively as a larger target size itself.</p>
+          <figure id="basic-target-size">
+            <img src="img/target-size-requirement.png" alt="Three square boxes next to each other, each 24px wide." />
+            <figcaption>Where targets are a 24 by 24px square (and larger is better), they meet the first line of the criterion and pass.</figcaption>
+          </figure>
+
+          <p>The square does not have to be aligned with the page, it could be skewed.</p>
+
+          <figure id="basic-target-size-skewed">
+            <img src="img/target-size-requirement3.png" alt="A skewed rectangle that includes a 24 by 24px square within it." />
+            <figcaption>So long as there is a 24 by 24px square within the target it meets the first line of the criterion and passes.</figcaption>
```

Comment:
Adding the word "solid" to avoid anyone interpreting that a hollow (or partially hollow) 24x24 outline would pass this...
```suggestion
            <figcaption>So long as there is a solid 24 by 24px square within the target, it meets the first line of the criterion and passes.</figcaption>
```